safe/gui/tools/options_dialog.py

```diff
@@ -107,48 +106,89 @@ def __init__(self, iface, dock=None, parent=None, qsetting=''):
             self.set_org_disclaimer)
 
     def restore_state(self):
-        """Reinstate the options based on the user's stored session info.
-        """
-        # flag = settings.value(
-        #     'inasafe/useThreadingFlag', False)
-        # hack set use thread to false see #557
-        # TODO: Store default values somewhere else, e.g. defaults.py
+        """Reinstate the options based on the user's stored session info."""
         flag = False
         self.cbxUseThread.setChecked(flag)
```

Comment:
I don't understand this purpose?
Dos that mean we never support threading in the future? Why  not delete the options altogether?